### ISSUE TITLE
[Suggestion | Bug] Incompatible EnviroMine HUDs with AdvancedHUD

### ISSUE THREAD

**Issue by Ac1dSn0w** [Funwayguy/EnviroMine#106]
[spoiler=Screenshot]

http://i.imgur.com/zjxlKgm.png

[/spoiler]

[Bug Reason] EnviroMine HUD visuals disappear for {thirst, temperature, sanity, oxygen} when AdvancedHUD mod is active.

[Suggestion Reason] Add compatibility AdvancedHUD mod to EnviroMine

Mod Used:
==========
 Link: https://mods.io/mods/1057-advancedhud

**Comment by Funwayguy** [comment]
Have you tried using the options to reposition the bars? Even if you can't see them originally the menu should still attempt to move them under the mouse.

**Comment by Ac1dSn0w** [comment]
Just tested it. Visuals are still buggy in-gaming. Here's more screenshots.

**Comment by Funwayguy** [comment]
Hmm... looking at AdvancedHUD code there may be a problem when the whole GUI is replaced, notably I don't see any Forge Events fired that EnviroMine needs to render it's GUI properly. It might be worth while to let the developer of AdvancedHUD (maxpowa) know about this and see what they can do their side.

PS: I've seen plenty of mods with HUD elements used along side EnviroMine with no issue so I assume our side is fine however I could still be wrong.

**Comment by GenDeathrow** [comment]
Yeah, @Funwayguy is right. Advance-HUD doesn't have the forge hooks which is what we need to render our HUD.  If they add the hooks in it will work. Advance-HUD is what I got my inspiration for our HUD. But using it they way we did made it so it's comparable with other mods.
